app/views/content/show.html.erb

```diff
@@ -4,4 +4,15 @@
 <h1 class="govuk-heading-l">
   <%= @page.title %>
 </h1>
-<%= translate_markdown(@page.markdown) %>
+<% if @page.content_list %>
+  <p class="govuk-body govuk-!-margin-bottom-1"> Contents </p>
```

Comment:
Can your confirm with @helenrappand or Bello that they want this heading?  I just note on the example on Prepared there is no heading for the contents list.

Reply:
Okay I will check with them, just thought to add it as the heading is there in the screenshot for the ticket